experimental/basisLieHighestWeight/LieAlgebras.jl

```diff
@@ -0,0 +1,50 @@
+# ?
+
+using Oscar
+using SparseArrays
+
+G = Oscar.GAP.Globals
+forGap = Oscar.GAP.julia_to_gap
+fromGap = Oscar.GAP.gap_to_julia
```

Comment:
Don't do that:
- you have this exact code in multiple files, that's bad
- it makes it very hard for others to read your code, as locally it is not clear what `G` stands for
- `gap_to_julia` shouldn't be used in production code, it is just for interactive use, as it is not type stable. Instead specify explicit target types, e.g. `Vector{Int}(gapobj)` to convert a GAP list of integers into a `Vector{Int}`, etc. (this is type stable and at the same time also verifies the GAP object has the right shape, and if not, throws an exception)
- don't use `julia_to_gap` either, instead use `GAP.Obj`